### PR TITLE
backend development

### PR DESCRIPTION
**Pull request by mandus**

### CODE REVIEW

**Comment by mandus** [comment]
It's not so easy to get a page like this right - there are a lot of stuff that we _can_ write about, but hard to tell what will resonate better with an intended audience (someone who likes to by services from us...). Consider this a very first iteration - glad if any of you have something to contribyte @eigilhs @stianlagstad @starsheriff or others who happen to read!

**Comment by skavhaug** [comment]
Great work at adding the contact! I think we should also reference the ppl in about us in the different topics covered in services. Its quite some additional work, but I'll get on with it in a separate branch tomorrow.

File: content/services/backend/index.md [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
+testing environments, continous integration and deployment in addition to the
+development environment itself.
+
+When it comes to the actual tools that are used in a project, we try to be
+pretty agnostic. Often there can be many tools that are right, and which one
+that is selected comes down to the preferences and competences with the client.
+We are fine with that, and can adapt, as we have through the years come to
+understand that a tool is often just syntax. We contribute with deeper
+understanding and ability to solve the problems at hand - and solutions can
+then be coded up in the environment at hand. There are however situations where
```
[the remark this answers is not in the document]
Comment (mandus) [reply content/services/backend/index.md]: In language I totally agree, but else it's nice with two hands :)

File: content/services/backend/index.md [hunk 2]
```diff
@@ -1,10 +1,42 @@
...
+developer or technical architect. And we are eager at keeping up with modern
+trends such as variants of agile development, test-driven development, and
+DevSecOps. As such, a narrow view on the programming environment is not
+sufficient, but rather a holistic view that includes database, run time and
+testing environments, continous integration and deployment in addition to the
+development environment itself.
+
+When it comes to the actual tools that are used in a project, we try to be
+pretty agnostic. Often there can be many tools that are right, and which one
+that is selected comes down to the preferences and competences with the client.
```
[the remark this answers is not in the document]
Comment (mandus) [reply content/services/backend/index.md]: What do you think of the new attempt?

[the remark this answers is not in the document]
Comment (skavhaug) [reply content/services/backend/index.md]: Better!

**Comment by eigilhs** [comment]
How set are we on the name? Could we call it something like "software engineering" instead?

**Comment by skavhaug** [comment]
I think this can be a good idea, and it does require some more rework of the text to fit into that category.

**Comment by mandus** [comment]
> How set are we on the name? Could we call it something like "software engineering" or "systems development" instead?

I like "systems development" (the other is unfortunately in my mind linked to activities that often don't include actual coding..). Are you willing to contribute an update to the PR with necessary changes @eigilhs , or do we ned to write it all over again with a new point of view?

**Comment by eigilhs** [comment]
> I like "systems development" (the other is unfortunately in my mind linked to activities that often don't include actual coding..).

Hm. I don't share that connotation, but I think I see what you mean. Although I think "software engineering" sounds like it has more of a basis i science. And it has the advantage that it's unambiguous even for non-technical people.

> Are you willing to contribute an update to the PR with necessary changes @eigilhs , or do we ned to write it all over again with a new point of view?

I'll try!

**Pull request merged**